README.md

```diff
@@ -1,24 +1,80 @@
-**STILL IN DEVELOPMENT, NOT FIT FOR GENERAL CONSUMPTION, DO NOT USE IF NOT A DEVELOPER!**
-
 # Slic3r plugin for OctoPrint
 
-**WARNING** Not functional yet since slic3r's return codes are behaving unexpectedly. WIP for others to take a look at
```

Comment:
I wrongly forgot to specify that this was tested under a Raspberry Pi (working versions are specified if I am not wrong), so I am not sure if it will work under Windows before me or someone else tests it.